docs/develop/01_sdk/04_integrate/04_vitejs.md

```diff
@@ -15,22 +15,113 @@ Bootstrap using the template of your choice and install KILT + NodeJS polyfills:
 
 <!-- TODO: Update the SDK version using the CI action triggered on SDK releases. -->
 
-```bash
-yarn create vite my-app --template react-ts
-cd my-app
-yarn add @kiltprotocol/sdk-js
-yarn add --dev @esbuild-plugins/node-globals-polyfill
-yarn add --dev @esbuild-plugins/node-modules-polyfill
-```
 
-Adjust the `vite.config.ts` file to activate the polyfills:
+<Tabs groupId="react-vuejs-choice">
+      <TabItem value="react" label="React" default>
+        <CodeBlock title=React>
+            ```bash
+              yarn create vite my-app --template react-ts
+              cd my-app
+              yarn add @kiltprotocol/sdk-js
+              yarn add --dev @esbuild-plugins/node-globals-polyfill
+              yarn add --dev @esbuild-plugins/node-modules-polyfill
+              yarn add --dev rollup-plugin-node-polyfills
+            ```
+        </CodeBlock>
+      </TabItem>
+      <TabItem value="vuejs" label="VueJS">
+        <CodeBlock title=VueJS>
+          ```bash
+            yarn create vite my-app --template vue-ts
+            cd my-app
+            yarn add @kiltprotocol/sdk-js
+            yarn add --dev @esbuild-plugins/node-globals-polyfill
+            yarn add --dev @esbuild-plugins/node-modules-polyfill
+            yarn add --dev rollup-plugin-node-polyfills
+          ```   
+        </CodeBlock>
+      </TabItem>
+    </Tabs>
 
-<CodeBlock className="language-ts">
-  {ViteConfig}
-</CodeBlock>
 
+Adjust the `vite.config.ts` file to activate the polyfills:
+<Tabs groupId="react-vuejs-choice">
+      <TabItem value="react" label="React" default>
+            <CodeBlock className="language-ts">
+                  {ViteConfig}
+            </CodeBlock>
+      </TabItem>
+      <TabItem value="vuejs" label="VueJS">
+        <CodeBlock title=VueJS className="language-ts">
+          ```
+            import vue from '@vitejs/plugin-vue'
```

Comment:
, We have our `ViteConfig` snippet inside `code_examples/vitejs/vite.config.ts`, which we then import here as a snippet, and should probably be renamed `ReactConfig`. You should do the same with the VueJS config, so that it is caught by our CI and tested for any breaking changes with our latest SDK version.